core/templates/pages/exploration-player-page/services/stats-reporting.service.ts

```diff
@@ -190,14 +190,14 @@ export class StatsReportingService {
     StatsReportingService.explorationStarted = true;
   }
 
-  recordExplorationActuallyStarted(stateName: string): void {
+  recordExplorationActuallyStartedAsync(stateName: string): void {
```

Comment:
Why is this async? It doesn't return anything? It does call an async function, but the issue was just to rename functions that returns it, right?